devel/ansible/roles/dev/tasks/main.yml

```diff
@@ -33,7 +31,6 @@
       state: present
   with_items:
       - gearbox
-      - crank==0.7.3
```

Comment:
I think that was the reason f28 failed. Since the verison of Turbogears in f28 requires crank >= 0.8.0. Could you give it another try ?

Reply:
I clean